grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -1,82 +1,113 @@
-import React, { FC, useCallback, useState } from 'react';
+import React, { FC, useCallback } from 'react';
 
-import { Button, Drawer, HorizontalGroup, VerticalGroup } from '@grafana/ui';
+import { css } from '@emotion/css';
+import { Button, Drawer, Field, HorizontalGroup, TextArea, useStyles2, VerticalGroup } from '@grafana/ui';
 import { observer } from 'mobx-react';
+import { Controller, FormProvider, useForm } from 'react-hook-form';
 
-import { GForm } from 'components/GForm/GForm';
 import { AddResponders } from 'containers/AddResponders/AddResponders';
 import { prepareForUpdate } from 'containers/AddResponders/AddResponders.helpers';
 import { AlertReceiveChannelStore } from 'models/alert_receive_channel/alert_receive_channel';
 import { ApiSchemas } from 'network/oncall-api/api.types';
 import { useStore } from 'state/useStore';
 import { openWarningNotification } from 'utils/utils';
 
-import { manualAlertFormConfig, ManualAlertGroupFormData } from './ManualAlertGroup.config';
+export type FormData = {
+  message: string;
+};
 
 interface ManualAlertGroupProps {
   onHide: () => void;
   onCreate: (id: ApiSchemas['AlertGroup']['pk']) => void;
   alertReceiveChannelStore: AlertReceiveChannelStore;
 }
 
-const data: ManualAlertGroupFormData = {
-  message: '',
-};
-
 export const ManualAlertGroup: FC<ManualAlertGroupProps> = observer(({ onCreate, onHide }) => {
   const { directPagingStore } = useStore();
   const { selectedTeamResponder, selectedUserResponders } = directPagingStore;
 
-  const [formIsValid, setFormIsValid] = useState<boolean>(false);
-
   const onHideDrawer = useCallback(() => {
     directPagingStore.resetSelectedUsers();
     directPagingStore.resetSelectedTeam();
     onHide();
   }, [onHide]);
 
+  const formMethods = useForm<FormData>({
+    mode: 'onChange',
+    defaultValues: { message: '' },
+  });
+
+  const {
+    handleSubmit,
+    control,
+    formState: { errors },
+  } = formMethods;
+
   const hasSelectedEitherATeamOrAUser = selectedTeamResponder !== null || selectedUserResponders.length > 0;
-  const formIsSubmittable = hasSelectedEitherATeamOrAUser && formIsValid;
+  const formIsSubmittable = hasSelectedEitherATeamOrAUser;
```

Comment:
These 2 in here can be 1 line instead and named directly to `canSubmitForm` or your naming